demo/demoserver.py

```diff
@@ -0,0 +1,64 @@
+'''Demoserver for RobotFlow
+'''
+from os.path import abspath
+import cherrypy
+from pathlib2 import Path
+
+
+class FormGenerator():
+    '''Main CherryPy class for showing pages
+    '''
+    def __init__(self):
+        self.valid_customers = ['345', '732', '834']
+        self.expired_customers = ['432', '545', '678']
+
+    @cherrypy.expose
+    def index(self):
+        '''Display main page
+        '''
+        # pylint: disable=no-self-use
+        return open("form/form.html")
+
+    @cherrypy.expose
+    def generate(self, **kwargs):
+        '''Show customerId status
+        '''
+        keywords = kwargs.keys()
+        if 'customerid' not in keywords:
+            raise Exception
```

Comment:
If I'm reading this correctly, this exception will not help the user in any way, though at least there will be a backtrace.